config/configure.in

```diff
@@ -563,6 +563,40 @@ else
   AC_MSG_RESULT([found $NUMS_INCLUDE])
 fi
 
+CURL=yes
+{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking ocaml curl library" >&5
+printf %s "checking ocaml curl library... " >&6; }
+CURL_INCLUDE=$(ocamlfind query -i-format curl 2> /dev/null)
+if [ $? -ne 0 ]; then
```

Comment:
These brackets are being removed by `autoconf` when generating `configure`. Better use `test $? -ne 0` instead. Same with the remaining ones bellow.